compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmCodeGenUtil.java

```diff
@@ -507,30 +511,70 @@ public static boolean isBallerinaBuiltinModule(String orgName, String moduleName
         return orgName.equals("ballerina") && moduleName.equals("builtin");
     }
 
-    public static void generateBbInstructions(MethodVisitor mv, LabelGenerator labelGen, JvmInstructionGen instGen,
-                                              int localVarOffset, AsyncDataCollector asyncDataCollector,
-                                              String funcName,
-                                              BIRNode.BIRBasicBlock bb) {
+    public static BirScope generateBbInstructions(MethodVisitor mv, LabelGenerator labelGen, JvmInstructionGen instGen,
```

Comment:
method name may not be correct as it returns BirScope?

Reply:
Fixed